hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/TssConfig.java

```diff
@@ -21,4 +21,5 @@ public record TssConfig(
         @ConfigProperty(defaultValue = "data/keys/tss") @NodeProperty String tssKeysPath,
         @ConfigProperty(defaultValue = "1024") @NetworkProperty int initialCrsParties,
         @ConfigProperty(defaultValue = "false") @NetworkProperty boolean hintsEnabled,
-        @ConfigProperty(defaultValue = "false") @NetworkProperty boolean historyEnabled) {}
+        @ConfigProperty(defaultValue = "false") @NetworkProperty boolean historyEnabled,
+        @ConfigProperty(defaultValue = "false") @NetworkProperty boolean forceHandoffs) {}
```

Comment:
We should probably mention when to set this to true and in which environments.

Reply:
Added a comment. 💯